src/main/java/com/dnd/runus/presentation/v1/oauth/OauthController.java

```diff
@@ -29,11 +33,30 @@ public class OauthController {
     @ApiErrorType({ErrorType.UNSUPPORTED_SOCIAL_TYPE, ErrorType.MALFORMED_ACCESS_TOKEN, ErrorType.UNSUPPORTED_JWT_TOKEN
     })
     @PostMapping
+    @ResponseStatus(HttpStatus.OK)
     public TokenResponse signIn(@Valid @RequestBody OauthRequest request) {
         return oauthService.signIn(request);
     }
 
+    @Operation(
+            summary = "회원 탈퇴",
+            description =
+                    """
+            ## 회원 탈퇴 API입니다.
+            idToken과 authorizationCode로 애플에서 토큰을 발급합니다.<br>
+            발급 받은 토큰으로 사용자의 서비스 해제 요청을 합니다.<br>
+            사용자의 모든 데이터를 삭제합니다.
+            """)
+    @ApiErrorType({
+        ErrorType.UNSUPPORTED_SOCIAL_TYPE,
+        ErrorType.UNSUPPORTED_JWT_TOKEN,
+        ErrorType.FAILED_AUTHENTICATION,
+        ErrorType.MALFORMED_ACCESS_TOKEN,
+        ErrorType.EXPIRED_ACCESS_TOKEN,
+        ErrorType.UNHANDLED_EXCEPTION,
+    })
```

Comment:
아래 2개 제외하고는 client가 무슨 에러인지 알아서 처리할 필요가 없을 것 같아요

```suggestion
    @ApiErrorType({
        ErrorType.UNSUPPORTED_SOCIAL_TYPE,
        ErrorType.FAILED_AUTHENTICATION,
    })
```